src/main/java/appeng/client/texture/WirelessTextures.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * This file is part of Applied Energistics 2. Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved. Applied
+ * Energistics 2 is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+ * later version. Applied Energistics 2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General
+ * Public License for more details. You should have received a copy of the GNU Lesser General Public License along with
+ * Applied Energistics 2. If not, see <http://www.gnu.org/licenses/lgpl>.
+ */
+
+package appeng.client.texture;
+
+import net.minecraft.client.renderer.texture.TextureMap;
+import net.minecraft.util.IIcon;
+import net.minecraft.util.ResourceLocation;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+public enum WirelessTextures {
```

Comment:
It might be nice to connect the on/off textures with eachother rather than separate. For example:
```java
ConnectorWhite("WirelessConnectorOnWhite", "WirelessConnectorOffWhite"),

// ...

HubWhite("WirelessHubOnWhite", "WirelessHubOffWhite"),

// ...
```
and then provide methods like `getOnIcon()` and `getOffIcon()`